script/conflate_osm.sh

```diff
@@ -10,7 +10,7 @@ DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd );
 # ---- pbf2json ----
 
 # location of pbf2json binary
-PBF2JSON_BIN=${PBF2JSON_BIN:-"pbf2json.linux-x64"};
+PBF2JSON_BIN=${PBF2JSON_BIN:-"$DIR/../node_modules/pbf2json/build/pbf2json.linux-x64"};
```

Comment:
oh that's a good catch. we should definitely fix that in the future and see if we can test it on a windows machine or something.